core/store/models/common.go

```diff
@@ -565,3 +565,9 @@ func (r *AddressCollection) Scan(value interface{}) error {
 	*r = collection
 	return nil
 }
+
+// Configuration stores key value pairs for overriding global configuration
+type Configuration struct {
+	Name  string `gorm:"not null;unique;index"`
```

Comment:
For simplicity's sake, do we want this to have an integer primary key? If not, shall we just make `name` the primary key? I feel uneasy about a table with no primary key.

As you know, embedding `gorm.Model` gives us an ID PK, and createdAt for little cost.